WebRoot/js/zimbraMail/abook/view/ZmContactSearch.js

```diff
@@ -428,23 +418,46 @@ ZmContactSearchListView.prototype._getHeaderList =
 function() {
 	var headerList = [];
 	headerList.push(new DwtListHeaderItem({field:ZmItem.F_TYPE, width:ZmMsg.COLUMN_WIDTH_FOLDER_CN}));
-	headerList.push(new DwtListHeaderItem({field:ZmItem.F_NAME, text:ZmMsg._name, width:ZmMsg.COLUMN_WIDTH_NAME_CN}));
-	headerList.push(new DwtListHeaderItem({field:ZmItem.F_EMAIL, text:ZmMsg.email}));
-
+	headerList.push(new DwtListHeaderItem({field:ZmItem.F_NAME, text:ZmMsg._name}));
 	return headerList;
 };
 
+// Overriding the DwtListView.prototype.createHeaderHtml as we donot need a header, here.
+
+ZmContactSearchListView.prototype.createHeaderHtml =
+function() {
+
+};
+
+ZmContactSearchListView.prototype._getCellClass =
+function (item, field) {
+	if (field == ZmItem.F_TYPE) {
+		return "ZmContactGroupItemImage";
+	} else if (field == ZmItem.F_NAME) {
+		return "ZmContactGroupItemContent";
+	}
+};
+
 /**
  * @private
  */
 ZmContactSearchListView.prototype._getCellContents =
 function(htmlArr, idx, contact, field, colIdx, params) {
 	if (field == ZmItem.F_TYPE) {
-		htmlArr[idx++] = AjxImg.getImageHtml(contact.getIcon());
+		var avatarUrl = contact.getImageUrl();
+		if (avatarUrl) {
+			htmlArr[idx++] = "<div class='ZmContactIcon'>";
+			htmlArr[idx++] = "<img src='" + avatarUrl + "' alt='" + contact.getFullName() + "' />";
+			htmlArr[idx++] = "</div>";
+		} else {
+			htmlArr[idx++] = AjxImg.getImageHtml(contact.getIcon());
+		}
 	} else if (field == ZmItem.F_NAME) {
+		htmlArr[idx++] = "<div><strong>";
```

Comment:
In that case its a issue right, as the email would also not be rendered because they are in the same if block.